site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -934,6 +942,33 @@ public function saveSingleComponent() {
         return $response;
     }
 
+    
+
+    public function ajaxGetStudentOutput() {
+
+        $gradeable_id = $_REQUEST['gradeable_id'];
+        $who_id = $_REQUEST['who_id'];
+        $count = $_REQUEST['count'];
+        $gradeable = $this->core->getQueries()->getGradeable($gradeable_id, $who_id);
+        
+
+        //Turns off the header and footer so that it isn't displayed in the testcase output
+        //Don't re-enable. 
+        $this->core->getOutput()->useHeader(false);
+        $this->core->getOutput()->useFooter(false);
+
+        $return = "";
+
+        $popup_css = "{$this->core->getConfig()->getBaseUrl()}css/diff-viewer.css";
```

Comment:
are we loading the same CSS for every student output? Is there any reason to not just load it once in the initial controller function that gets fired?

Reply:
This is an artifact of the way Andrew M. wrote the function before. It looks like the openPopUp function in server.js takes in a css file path as input, which it then uses to style the resulting popup window. This css is only loaded if the user chooses to open a popup version of their output.